tensorflow/contrib/kernel_methods/python/kernel_estimators_test.py

```diff
@@ -92,7 +92,7 @@ def testLinearlySeparableBinaryDataNoKernels(self):
     # Since the data is linearly separable, the classifier should have small
     # loss and perfect accuracy.
     self.assertLess(metrics['loss'], 0.1)
-    self.assertEqual(metrics['accuracy'], 1.0)
+    self.assertAlmostEqual(metrics['accuracy'], 1.0, 7)
```

Comment:
According to the comment above, I am worried about almost equal here. I know 7 digits is very very precise, but still we are making the test looser.